models/migrations/migrations.go

```diff
@@ -82,6 +82,8 @@ var migrations = []Migration{
 	NewMigration("create user column allow create organization", createAllowCreateOrganizationColumn),
 	// V16 -> v17
 	NewMigration("create repo unit table and add units for all repos", addUnitsToTables),
+	// v17 -> v18
+	NewMigration("set protect branches updated with created", setProtectedBranchUpdatedWithCreated),
```

Comment:
It's an entirely new struct, that doesn't need any migration because of the anyway executed sync function